repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/Valve.scala

```diff
@@ -127,10 +127,9 @@ class Valve {
 
         //Set up all the valve context, it's lazy, so it won't get instantiated until here as well.
         valveContext.setParent(coreContext)
-        valveContext.scan("org.openrepose.valve.spring") //TODO: config file?
+        valveContext.scan("org.openrepose.valve") //TODO: config file?
```

Comment:
To pick up the class in `org.openrepose.valve` (the Jetty hooks that call into the logging service). Granted, I could have nested those classes somewhere in the `spring` package, but that didn't feel quite right to. Especially since the `org.openrepose.valve.jetty` package already exist.

Actually, I was a little careless here. This change will also cause this scan to pick up `ValvePortMXBeanImpl` which seems like it is being created manually currently (despite being annotated) when running in `test-mode`. However, the code to set up that bean includes the following comment:
> //If I'm in test mode, set up the port mbean... Heck, can probably always set it up

Oddly, a quick test shows that scanning this bean does not in fact set it up with the JMX. I am really not sure why that is the case, but I suspect we were having problems getting Spring to set up this JMX bean and that is why we ended up doing manually. That being the case, scanning this bean likely only creates a nearly empty object and nothing else. Still not ideal, but I can live with it. Especially since it would probably be more ideal to manage this bean with Spring (in which case scanning it would be warranted), we seemingly just did not get that working.